lmdeploy/serve/openai/api_server.py

```diff
@@ -389,6 +393,15 @@ async def chat_completions_v1(request: ChatCompletionRequest,
         1.0 means no penalty
     - stop (str | List[str] | None): To stop generating further
         tokens. Only accept stop words that's encoded to one token idex.
+    - tools (List): A list of tools the model may call. Currently, only
+        internlm2 functions are supported as a tool. Use this to provide
+        a list of functions the model may generate JSON inputs for.
```

Comment:
`Use this to provide a list of functions the model may generate JSON inputs for` sounds a bit confusing.

[Grammarly](https://app.grammarly.com/) gives `Use this to specify a list of functions for which the model can generate JSON inputs`